client/src/app/containers/Game/CutAction.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+
+import Button from '@material-ui/core/Button';
+import Slider from '@material-ui/core/Slider';
+import CallSplitIcon from '@material-ui/icons/CallSplit';
+
+import { ActionInputProps } from './types';
+import { useGame } from './useGame';
+
+const CutAction: React.FunctionComponent<ActionInputProps> = ({
+    isBlocking,
+}) => {
+    const [percentage, setPercentage] = useState(50);
```

Comment:
:mag: Should this be `cutPercentage, setCutPercentage`?

Reply:
Same comment as above: if you think adding the word cut adds significant value to someone reading this code, I'm open to it. When I wrote this, my thought was _probably_ that since we're within `CutAction`, it's fairly obvious what `percentage` means.